src/ShopifyApp/Actions/CreateWebhooks.php

```diff
@@ -45,13 +44,13 @@ public function __invoke(ShopIdValue $shopId, array $configWebhooks): array
          * Checks if a webhooks exists already in the shop.
          *
          * @param array $webhook  The webhook config.
-         * @param ResponseAccess $webhooks The current webhooks to search.
```

Comment:
@onurkose Is array correct? Because ResponseAccess is what is returned from the API call to get the webhook list. It has array access support but its actually a class.

Reply:
@osiset, No it was not, but it's correct now.